src/test/java/org/broadinstitute/consent/http/db/UserDAOTest.java

```diff
@@ -47,38 +55,38 @@ public void testDescribeUsersByRole() {
         createUserWithRole(UserRoles.MEMBER.getRoleId());
 
         List<DACUser> members = userDAO.describeUsersByRole(UserRoles.MEMBER.getRoleName());
-        Assert.assertFalse(members.isEmpty());
+        assertFalse(members.isEmpty());
 
         List<DACUser> chairs = userDAO.describeUsersByRole(UserRoles.CHAIRPERSON.getRoleName());
-        Assert.assertFalse(chairs.isEmpty());
+        assertFalse(chairs.isEmpty());
 
         // Only case where we don't set up users by default.
         List<DACUser> alumni = userDAO.describeUsersByRole(UserRoles.ALUMNI.getRoleName());
-        Assert.assertTrue(alumni.isEmpty());
+        assertTrue(alumni.isEmpty());
 
         List<DACUser> admins = userDAO.describeUsersByRole(UserRoles.ADMIN.getRoleName());
-        Assert.assertFalse(admins.isEmpty());
+        assertTrue(admins.isEmpty());
```

Comment:
This change was necessary after removing Moran from `changelog-consent-10.0.xml`